proto/common.proto

```diff
@@ -518,6 +518,7 @@ message RaidBuffs {
 	TristateEffect stoneskin_totem = 24;
 	TristateEffect retribution_aura = 25;
 	bool sanctity_aura = 36;
+	int32 battle_squawk = 37;
```

Comment:
This is technically a party buff. A lot of our buffs are mis-labeled as raid buffs when it should be a party buff. It only _really_ matters for the raid sim and I don't think any of us have the time to maintain that for SoD so up to you if you want to change it

Reply:
Yea, a lot is hold over from raid wide in Wrath with LOTP/Auras etc all party in SoD.  I think its fine for you to merge as is, and if the raid sim every gets to fruition will need to remap every buff anyways.